portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase

```diff
@@ -0,0 +1,526 @@
+	@priority = 5
+	test TheLinkActionFieldsCannotBeEmpty {
+		task ("And creates a new item action") {
+			DataSetAdmin.addActions(
+				actionName = "Edit",
+				actionTab = "Creation Actions",
+				icon = "pencil",
+				readlessAction = "xx-ee",
+				type = "Link",
+				url = "https://gist.github.com/deborasoliveira/40716d0b03d33ff7e636ca5d366c79c5#file-link-action-url-example");
+		}
+
+		task ("When the user clicks on the new item action 3 dots button and edit") {
+			Click(
+				key_index = 2,
+				key_text = "Actions",
+				locator1 = "DataSet#VERTICAL_ELLIPSIS_N");
+
+			Click(
+				key_menuItem = "Edit",
+				locator1 = "MenuItem#DROPDOWN_MENU_ITEM");
+		}
+
+		task ("And go to Edit page for the action item created and leave the required fields, title and url empty") {
+			Type(
+				key_unnamedTitle = "",
+				locator1 = "TextInput#DEPOT_UNNAMED_DEFAULT_TITLE",
+				value1 = "");
+
+			Type(
+				key_fieldId = "URL",
+				locator1 = "TextArea#ANY_ID",
+				value1 = "");
+		}
+
+		task ("Then Error messages are displayed and the save button is disabled") {
+			AssertTextEquals(
+				locator1 = "Message#ERROR_FORM_FIELD_REQUIRED",
+				value1 = "This field is required.");
+
+			AssertElementPresent(locator1 = "DataSet#ANY_BUTTON_DISABLED");
+		}
+	}
+
+	@description = "LPS-194351 Confirm that when editing an already created modal action, it is not possible to leave the fields empty"
+	@priority = 5
+	test TheModalActionFieldsCannotBeEmpty {
+		task ("And clicks on Create Item Action button") {
+			DataSetAdmin.addActions(
+				actionName = "Edit",
+				actionTab = "Creation Actions",
+				icon = "pencil",
+				inputText = "Products",
+				type = "Modal",
+				url = "https://gist.github.com/deborasoliveira/40716d0b03d33ff7e636ca5d366c79c5#file-link-action-url-example",
+				variant = "Full Screen");
+		}
+
+		task ("And clicks on the Edit option") {
+			Click(
+				key_index = 2,
+				key_text = "Actions",
+				locator1 = "DataSet#VERTICAL_ELLIPSIS_N");
+
+			Click(
+				key_menuItem = "Edit",
+				locator1 = "MenuItem#DROPDOWN_MENU_ITEM");
+		}
+
+		task ("And go to Edit page for the action item created and leave the required fields, label, title and url empty") {
+			Type(
+				key_fieldName = "add-here-the-title-of-the-modal",
+				locator1 = "DataSet#PRESELECT_VALUE_INPUT",
+				value1 = "");
+
+			Type(
+				key_unnamedTitle = "",
+				locator1 = "TextInput#DEPOT_UNNAMED_DEFAULT_TITLE",
+				value1 = "");
+
+			Type(
+				key_fieldId = "URL",
+				locator1 = "TextArea#ANY_ID",
+				value1 = "");
+		}
+
+		task ("Then Error messages are displayed and the save button is disabled") {
+			AssertTextEquals(
+				locator1 = "Message#ERROR_FORM_FIELD_REQUIRED",
+				value1 = "This field is required.");
+
+			AssertElementPresent(locator1 = "DataSet#ANY_BUTTON_DISABLED");
+		}
+	}
+
+	@description = "LPS-194351 Confirm that a created action can be edited, and options are displayed"
+	@priority = 3
+	test TheNewActionCanBeEdited {
```

Comment:
You can remove this test. I've lowered this scenario to 2 in the story, so no automation test is needed.